### DISCUSSION TITLE
Move database and table description

### DISCUSSION THREAD

**Discussion by bendnorman** [simonw/datasette#1997] [Q&A]
Descriptions of tables and databases are available when you click on the [db](https://data.catalyst.coop/pudl) or [table](https://data.catalyst.coop/pudl/generation_fuel_eia923). Can the descriptions be added to the browsing pages so users can read descriptions without clicking on the dbs or tables? For example, all of the table descriptions would be viewable on the [database page](https://data.catalyst.coop/pudl).

**Comment by zaneselvans** [comment]
I think the way to implement this is to define a [custom template for the pages](https://docs.datasette.io/en/stable/custom_templates.html#custom-templates) using Jinja, inheriting from or overriding the default page templates that ship with Datasette.
